### DISCUSSION TITLE
2D transfer function editor improvements

### DISCUSSION THREAD

**Discussion by mlavik1** [mlavik1/UnityVolumeRendering#80] [Ideas]
## 2D Transfer function editor improvements

The current 2D transfer function editor is really bad, and I want to improve it.

If anyone have suggestions for improvements, please comment here.
Screenshots from other software that have a good 2D TF editor are also welcome.

**Things I want to improve:**
- More shapes (polygons?)
- Make it easier to move the shapes (currently you have to use multiple sliders)

See related task: #65

**Comment by vahpy** [comment]
I'm not very familiar with UI design flexibility in Unity, so my suggestion could be difficult to implement. Anyway, what I think a proper 2D TF editor need a layer management, including sort, hide, add and remove layers.
For drawing, I suggest to have at least a basic brush and eraser.

**Reply by mlavik1** [reply]
Sorry, I didn't see your message before now! Thanks for the suggestion :) That's a good idea. Do you know of any other software I could look at for comparison?

I'm hoping to have a look at this soon!
